client/src/elm/Pages/NCD/RecurrentActivity/Update.elm

```diff
@@ -0,0 +1,564 @@
+module Pages.NCD.RecurrentActivity.Update exposing (update)
+
+import App.Model
+import AssocList as Dict
+import Backend.Entities exposing (..)
+import Backend.Measurement.Model exposing (IllnessSymptom(..), ViralLoadStatus(..))
+import Backend.Measurement.Utils exposing (..)
+import Backend.Model exposing (ModelIndexedDb)
+import Backend.NCDActivity.Model
+import Backend.NCDEncounter.Model
+import EverySet exposing (EverySet)
+import Gizra.NominalDate exposing (NominalDate)
+import Gizra.Update exposing (sequenceExtra)
+import Maybe.Extra exposing (isJust, isNothing, unwrap)
+import Measurement.Utils
+    exposing
+        ( toRandomBloodSugarResultsValueWithDefault
+        , toSendToHCValueWithDefault
+        , toUrineDipstickResultsValueWithDefault
+        )
+import Pages.GlobalCaseManagement.Utils exposing (prenatalLabsResultsTestData)
+import Pages.NCD.RecurrentActivity.Model exposing (..)
+import Pages.NCD.RecurrentActivity.Utils exposing (..)
+import Pages.NCD.Utils exposing (..)
+import Pages.Page exposing (Page(..), UserPage(..))
+import Pages.Utils exposing (nonAdministrationReasonToSign, setMultiSelectInputValue)
+import RemoteData exposing (RemoteData(..))
+import Translate exposing (Language, translate)
+
+
+update : Language -> NominalDate -> NCDEncounterId -> ModelIndexedDb -> Msg -> Model -> ( Model, Cmd Msg, List App.Model.Msg )
+update language currentDate id db msg model =
+    let
+        noChange =
+            ( model, Cmd.none, [] )
+
+        generateLabResultsMsgs nextTask =
+            Maybe.map (\task -> [ SetActiveLabResultsTask task ]) nextTask
+                |> Maybe.withDefault [ SetActivePage <| UserPage <| NCDRecurrentEncounterPage id ]
+
+        -- generateNextStepsMsgs personId nextTask =
```

Comment:
Remove or `@todo`

Reply:
This all has been implemented at #456 and #459 already.